src/components/common/CommentView.js

```diff
@@ -411,9 +514,104 @@ class CommentView extends Component {
     );
   }
 
+  renderReactions() {
+    const { reactions } = this.props;
+    // if there are no reactions show nothing
+    if (reactions.every((g) => g.count === 0)) {
+      return null;
+    }
+    return (
+      <Grid container item direction="row" spacing={1} alignItems="center">
+        <Grid item>{this.renderAddReactions('small')}</Grid>
+        {reactions.map((reaction) =>
+          reaction.count ? (
+            <Grid item key={reaction.label}>
+              <Chip
+                label={`${reaction.icon} ${reaction.count}`}
+                size="small"
+                color="primary"
+                // here we use undefined because when using "filled" it does not work
```

Comment:
Confirm that you can use `contained`.

Reply:
In MUI v4, the variant property is called `'default'` to get the "filled" appearance.